_posts/2016-12-02-recurse-center-week-9.md

```diff
@@ -0,0 +1,167 @@
+
+Such a lovely mélange of socializing this week!
+
+### Intellectual
+
+#### **Building communities**
+
+I had a lot of great conversations throughout the week about *"What comes after RC?"*.
+
+Another idea that's gotten me excited is starting a coliving space. I spent most of Saturday:
+
+- Asking people about past experiences with coops.
+- Researching existing coops/coliving spaces.
+- Aggregating lists of resources.
+- Debating ideas with friends.
+
+I've started looking for spaces to visit or apply to. One consistent piece of advice was to give it a try before diving in.
+
+Nevertheless, questions around community building have been fascinating me and it seems like a stream of enthusiasm worth following.
+
+#### **How to spend my time?**
+
+Another question I've been pondering is *"How do I want to spend my time?"*.
+
+My one approach to making time decisions, is *"What are my goals? Is this the most impactful way to move towards them?"*.
+It's interesting how this changes the way I spend time.
+
+**One example:**
+
+I used to read a lot online. In fact, [EpubPress](https://epub.press) was created to help me manage my intake of reading material.
+
+- Quora
+- HackerNews
+- Newspapers
+- etc.
+
+I've since realized the majority of what I read isn't immediately useful to me.
+
+> "How we scale @ {insert_startup}"  
+> Ooo! How DO they scale? 😱
+
+*I don't know... but why does it matter? I see no need to create a hyper growth system any time soon. If I ever do, I'm sure there's better resources than something from HackerNews.*
+
+> "Introducing {insert_generic_word}. A simple, elegant and intuitive JS framework"  
+> Ooo, I write Javascript! I haz need for this 😱!
+
+*No you don't, Harold. You're in no need of a framework... in fact, nobody writing Javascript needs another framework.*
+
+I've generally found this pushing my time towards more fulfilling activities.
+
+#### **Haskell Progress**
+
+The other day I finished my CIS194 course! 🎓
+
+By the end of it, I really started enjoying Haskell. It's so different from Javascript and required learning a whole other set of concepts.
+
+I think it was the right choice 😊.
+
+### Physical
+
+#### **So much sleep!**
+
+The Hackathon last week in addition to the night-owling really messed up my sleep schedule. On Wednesday night I got home at 3am and cooked myself dinner...
+
+Getting things back on schedule so I can live a more normal life again. @hsubox also brought me a pillow, so goodbye pillowcase stuffed with sweater! 😁
+
+#### **Pending baking fiasco**
+
+I cooked Chicken Parmesan again last week!
+
+I also bought 2.25 kilograms of flour a month ago because my Eggplant Parmesan recipe called for a 1/2 cup of flour...so I'm currently rolling in excess flour.
+
+Decided to just go all in and buy a bunch of cookie ingredients. So hopefully no wasted flour! Look at me, making sure none of my $2.50 bag of flour is wasted by buying an additional $10 of ingredients. #Smart 🤓
```

Comment:
Will you be making [these](http://sweets.seriouseats.com/2013/12/the-food-lab-the-best-chocolate-chip-cookies.html)?